packages/nodes-base/nodes/HaloPSA/descriptions/ClientDescription.ts

```diff
@@ -0,0 +1,74 @@
+import { INodeProperties } from 'n8n-workflow';
+
+export const clientDescription: INodeProperties[] = [
+	{
+		displayName: 'Name:',
+		name: 'clientName',
+		type: 'string',
+		default: '',
+		description: 'Enter client name',
+		required: true,
+		displayOptions: {
+			show: {
+				operation: ['create', 'update'],
+				resource: ['client'],
+			},
+		},
+	},
+	// Additional fields =============================================================
+	{
+		displayName: 'Additional Fields',
```

Comment:
Additional fields are called when creating but then updating it should be called Update Fields. Check the Jira node for a reference.